care/facility/models/tests/test_patient.py

```
@@ -65,3 +65,46 @@ def test_year_of_birth_validation(self):
+            "year_of_birth": 2000,
+            "disease_status": "NEGATIVE",
+            "emergency_phone_number": "+919000000666",
+            "is_vaccinated": "false",
+            "number_of_doses": 0,
+            "phone_number": "+919000044343",
+            "is_antenatal": False,
+        }
+        self.client.force_authenticate(user=dist_admin)
+        response = self.client.post("/api/v1/patient/", sample_data, format="json")
+        if response.status_code != status.HTTP_201_CREATED:
+            print(response.data)
```

Ruff

`print` found

(T201)